spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -585,4 +585,30 @@
       expect(presenter.as_json[:attributes][:recruited_at]).to be_present
     end
   end
+
+  describe 'compliance with models that change updated_at' do
+    it 'looks at all fields which cause a touch' do
+      choice = create(:application_choice, :awaiting_provider_decision)
```

Comment:
We still need to add enough application choice types here to cover different cases in the application presenter